docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,344 @@
+
+### `displayName (String)`
+
+The human-friendly name of the image.
+
+### `version (String)`
+
+The version of the image.
+
+### `url (String)`
+
+The main url of the image.
+
+### `releaseNotesUrl (String)`
+
+The url to the image's release notes or CHANGELOG.
+
+### `logoPath (String)`
+
+The path to the image specific logo in SVG format.
+
+The SVG format requirement ensures that the logo looks good in all types of
+screen resolutions.
+
+### `checksumType (String)`
+
+The checksum type. The current possible values are: `sha1`, `crc32`, and `md5`.
+
+### `checksum (String)`
+
+The actual checksum according to the type declared in `checksumType`.
+
+### `bytesToZeroOutFromTheBeginning (Number)`
+
+The number of bytes to zero out from the beginning before starting the flash
+process. This option currently only applies when using bmaps.
+
+### `recommendedDriveSize (Number)`
+
+The recommended drive size to flash this image, in bytes.
+
+The use case for this option is that while a drive might be large enough to
+contain the image, it might not be large enough to deliver a good experience
+when actual using the application or operating system contained in the image.
+
+### `bmapPath (String)`
+
+Path to a [bmap][bmap] file.
+
+### `instructionsPath (String)`
+
+Path to a markdown file showing instructions to the user for when the flash
+completes.
+
+### `supportedDevices (String[])`
+
+An array of the device slugs this image supports.
+
+### `updateUrl (String)`
+
+The url where the client can request the latest version of the image.
+
+The way this works is that you provide a generic url that redirects to the
+latest image resource. The client will follow all redirects until it finds the
+final image, and will check if the resource matches the currently selected
+image.
+
+### `etag (String)`
+
+The [HTTP Etag][etag] of the current image.
+
+This can be used to check if the image resolved by `updateUrl` is different to
+the one we have locally.
+
+### `expiryDate (String)`
+
+The timestamp to determine at which point the image may be expired. The date
+should conform to [ISO 8601][iso8601] standard.
+
+Clients may use this information to refuse flashing the image.
+
+Client
+------
+
+> A client represents the program making use of the information about the other
+  entities. This is not something publishers declare themselves.
+
+_At the moment of this writing, the only existent client is the Etcher
+application._
+
+Here's an example of a real-world client manifest for Etcher running on Mac OS
+X:
+
+```json
+{
+  "name": "etcher",
+  "version": "1.0.0-beta.14",
+  "os": "darwin",
+  "arch": "x64"
+}
+```
+
+A client may be described with any of the following properties:
+
+### `name (String)`
+
+The client name.
+
+### `version (String)`
+
+The client version.
+
+### `os (String)`
+
+The operating system running the client.
+
+The current possible values are: `aix`, `darwin`, `freebsd`, `linux`,
+`openbsd`, `sunos`, and `win32`.
```

Comment:
Perhaps there should also be a Client `os_version` field?
(and again, this document should specify the allowed values, to stop different publishers using slightly different values to mean the same thing)